client/components/editor/teaching-elements/Video.vue

```diff
@@ -20,13 +20,18 @@
         </div>
       </div>
       <div class="player">
-        <video-player
-          v-if="!isDragged"
-          :options="options"
-          @ready="onReady"
-          ref="video"
-          class="player">
-        </video-player>
+        <plyr
+          v-if="!switchingVideo"
```

Comment:
I'm guessing this has something to do with Youtube/Vimeo embed? :thinking: 
Btw, you are sure that dragging embeds `v-if="!isDragged"` does not cause problems anymore (in Chrome)?